src/main/java/org/folio/fqm/service/QueryManagementService.java

```diff
@@ -207,8 +216,14 @@ private List<Map<String, Object>> getContents(UUID queryId, UUID entityTypeId, L
           .map(FqlField::getColumnName)
           .collect(Collectors.toList());
       }
-      if (!fields.contains("id")) {
-        fields.add("id");
+      List<String> idColumnNames = entityTypeService
+        .getEntityTypeDefinition(entityTypeId)
+        .map(IdColumnUtils::getIdColumnNames)
+        .orElseThrow(() -> new EntityTypeNotFoundException(entityTypeId));
+      for (String idColumnName : idColumnNames) {
+        if (!fields.contains(idColumnName)) {
+          fields.add(idColumnName);
+        }
```

Comment:
This is a holdover from the composite id ticket. I missed it because I wasn't thinking about the synchronous GET /query API.